chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -242,10 +242,6 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
         // Toggle chat visibility
         BroadcastService.getMessageByEventName(BroadcastEvent.HideChatVisibilityChangeEvent).subscribe(async (event) => {
             if (event?.payload?.isChatHidden !== undefined) {
-                TelemetryHelper.logActionEvent(LogLevel.INFO, {
```

Comment:
Why are we removing this telemetry ?

Reply:
We're using the script layer telemetry event. This would just be noise⁠